sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 1.0.0-beta.5 (2021-03-09)
```

Comment:
let's make sure we align the versions across the packages. I see some of them are beta.6.

Reply:
What's the process for this? Do I just bump the version from to 6 and skip 5?
or just immediately make a 6 with 1 change of increment version number?

Reply:
oh i misunderstood this and thought it was just for chat? is this for all the JS comms packages?